src/lava/lib/dl/bootstrap/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2021 Intel Corporation
+# Copyright (C) 2022 Intel Corporation
```

Comment:
Do we for sure need to do this every year?

Reply:
If these are new files we need to change copyright. 

Otherwise we should update to a date range when they are touched not be default.